source/crud/crud.md

```diff
@@ -712,6 +714,8 @@ class FindOptions {
    */
   let: Optional<Document>;
 }
+
+type FindOneOptions = Omit<FindOptions, 'batchSize' | 'cursorType' | 'limit' | 'noCursorTimeout'>;
```

Comment:
In "findOne API details" below, you talk about how `singleBatch` must be set to `true`. Shouldn't that option also be omitted here?

Looking at PHPLIB, we don't expose that in the `Collection::findOne()` API.

Reply:
Oh, I forgot that `singleBatch` isn't even in FindOptions (and there's a Q&A entry addressing that). Carry on.